src/Server.UI/Components/Stepper/StatusResultStep.razor

```diff
@@ -1,10 +1,14 @@
-﻿<MudStepExtended IsResultStep="true">
+﻿@using System.ComponentModel
+<MudStepExtended IsResultStep="true">
 	<MudItem Class="d-flex justify-center">
 		@if (SubmissionSuccess)
 		{
-			<MudStack Row="true">
-				<MudText Typo="Typo.h4">@SuccessText</MudText>
-				<MudIcon Size="Size.Large" Class="align-self-center" Icon="@Icons.Material.Filled.Check" Color="Color.Success" />
+			<MudStack Row="false">
+				<MudText Typo="Typo.h4">Assessment Submitted</MudText>
```

Comment:
We should render `@SuccessText` here instead, as this is now a generic component.

```suggestion
				<MudText Typo="Typo.h4">@SuccessText</MudText>
```